src/messages.js

```diff
@@ -30,3 +30,8 @@ exports.newIssue = (username: string) => stripIndent`
     community that typically always has someone willing to help. You can sign-up [here](https://slack.babeljs.io/)
     for an invite.
 `;
+
+exports.tweetIssue = (issueURL: string) => stripIndent`
+    [issue] Beginner friendly issue for new contributors,
```

Comment:
I'll change the message to `If you're looking to contribute to Babel, we have another beginner-friendly issue for you ! [issue link]`